pkg/controller/plan/adapter/ovirt/validator.go

```diff
@@ -165,3 +165,8 @@ func (r *Validator) MaintenanceMode(_ ref.Ref) (ok bool, err error) {
 	ok = true
 	return
 }
+
+// NO-OP
+func (r *Validator) StaticIPs(vmRef ref.Ref) (bool, error) {
+	return true, nil
```

Comment:
here we can comment that the guest operating system is not modified during the migration so static IPs should be preserved (also for OpenStack above)